waiter/integration/waiter/health_check_test.clj

```diff
@@ -85,6 +86,21 @@
       (run-ping-service-test waiter-url idle-timeout command backend-proto health-check-proto num-ports health-check-port-index
                              :query-params {"exclude" "service-state"}))))
 
+(deftest ^:parallel ^:integration-fast test-invalid-backend-proto-health-check-proto-combo
+  (testing-using-waiter-url
+    (let [supported-protocols #{"http" "https" "h2c" "h2"}]
+      (doseq [backend-proto supported-protocols]
+        (doseq [health-check-proto (disj supported-protocols backend-proto)]
+          (doseq [health-check-port-index [nil 0]]
```

Comment:
The bindings block for `doseq` works just like `for`, which means these 3 `doseq`s can be combined into a single block.

Reply:
Done.